src/components/Tooltip/index.js

```diff
@@ -93,15 +93,6 @@ export default class Tooltip extends PureComponent {
     return data
   }
 
-  applyStyle = (data) => {
```

Comment:
was this supposed to be copied and not deleted?

Reply:
somehow it was a duplicate of https://github.com/yankaindustries/mc-components/pull/497/files/a705c179d6acce7c7b8ae7c9b049abf7a5457d66#diff-0ad95356130ae09835069df6f53fbdbeL87